cfn-resources/global-cluster-config/cmd/resource/resource.go

```diff
@@ -0,0 +1,376 @@
+package resource
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+
+	"github.com/spf13/cast"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	"github.com/openlyinc/pointy"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var RequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ClusterName, constants.ProjectID}
+
+func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler.ProgressEvent, error) {
+	setup() // logger setup
+
+	_, _ = logger.Debugf("Create snapshot for Request() currentModel:%+v", currentModel)
+
+	// Validate required fields in the request
+	if modelValidation := validateModel(RequiredFields, currentModel); modelValidation != nil {
+		return *modelValidation, errors.New("required field not found")
+	}
+
+	// Create MongoDb Atlas Client using keys
+	client, err := util.CreateMongoDBClient(*currentModel.ApiKeys.PublicKey, *currentModel.ApiKeys.PrivateKey)
+	if err != nil {
+		_, _ = logger.Warnf(constants.ErrorCreateMongoClient, err)
```

Comment:
we can remove the _, _ = is not needed
```suggestion
		logger.Warnf(constants.ErrorCreateMongoClient, err)
```

Reply:
i am getting lint errors if _,_= is removed from the code.All other modules is alsomaintaining _,_=